docs/rules/toggle-button-needs-labelling-v9.md

```diff
@@ -0,0 +1,40 @@
+# Accessibility: Toggle buttons must have accessible labelling: title, aria-label, aria-labelledby, aria-describedby (`@microsoft/fluentui-jsx-a11y/toggle-button-needs-labelling-v9`)
+
+<!-- end auto-generated rule header -->
+
+All interactive elements must have an accessible name.
+
+Toggle buttons without additional text content lack an accessible name.
+
+Please add title, aria-label, aria-labelledby, aria-described by etc.
+
+<https://www.w3.org/TR/html-aria/>
+
+## Rule Details
+
+This rule aims to prevent an icon button from not having an accessible name.
+
+Examples of **incorrect** code for this rule:
+
+```jsx
+<ToggleButton/>
```

Comment:
Buttons without text or an icon are actually not accessible to sighted users. Let's make sure the documentation reflects that. e.g.

correct
  
```
  <ToggleButton >
    Example
  </ToggleButton>
  
   <ToggleButton
    icon={<CalendarMonthRegular />}
  >
    Example
  </ToggleButton>
   
       <Tooltip content="With calendar icon only" relationship="label">
        <ToggleButton icon={<CalendarMonthRegular />} />
      </Tooltip>
	  
  <ToggleButton
    icon={<CalendarMonthRegular />}
    aria-label="Monthly calendar"
  >
  </ToggleButton>
	    
    <Label id="calendar-1">Compound example</Label>
    <ToggleButton aria-labelledby="calendar-1" icon={<CalendarMonthRegular />} />
```
  
  incorrect
  
```
    <ToggleButton></ToggleButton>
  
  
    <ToggleButton icon={<CalendarMonthRegular />}></ToggleButton>
	
	
	<Label>Compound example</Label>
    <ToggleButton icon={<CalendarMonthRegular />} />
```